### PR TITLE
state: Add the module information to the State

### LINKED ISSUE #240: Generated State does not have the Module on it.
**General information:**

* Operating System: 
* Terracognita version / tag:
* Did you build Terracognita from sources or did you use the Docker image: 

**Describe the bug:**

When using modules the State does not have the module on it, the resources do not have it declared.

**Log message**

Here you can paste the log message or paste the link to console logs. If the log message is too big, you can use a tool like https://pastebin.com/.

**Additional context**

### PR DESCRIPTION
**Pull request by xescugc**
Now the resources will have the module information on it if it's imported as a module

Closes #240

### CODE REVIEW

File: state/writer_test.go [hunk 1]
```diff
@@ -220,6 +220,85 @@ func TestSync(t *testing.T) {
...
+                  "unique_id":null
+               },
+               "schema_version":0
+            }
+         ],
+         "mode":"managed",
+         "name":"name",
+				 "module": "module.cycloid",
+      	 "provider": "provider[\"registry.terraform.io/hashicorp/aws\"]",
+         "type":"aws_iam_user"
```
Comment (kerak19) [inline state/writer_test.go:258]: The formatting looks wonky, shouldn't these `k:v` have the same indentation?

File: state/writer_test.go [hunk 2]
```diff
@@ -368,6 +447,151 @@ func TestDependencies(t *testing.T) {
...
+			state  = `{
+  "version": 4,
+  "terraform_version": "0.13.5",
+  "serial": 0,
+  "lineage": "lineage",
+  "outputs": {},
+  "resources": [
+    {
+      "mode": "managed",
+			"module": "module.cycloid",
```
Comment (kerak19) [inline state/writer_test.go:470]: Same as above, the indentation looks wrong (GH bug maybe?)

**Comment by xescugc** [comment]
ok fixed, it's some tabs/spaces, on my editor they look fine but on GH are different.

**Review by kerak19 — APPROVED** [review]
LGTM RS

**Pull request merged**